### ISSUE TITLE
List all the projects that are related to sovereign identities

### ISSUE THREAD

**Issue by satazor** [ipfs-shipyard/pm-idm#3]
There are a lot of projects working on sovereign identity, from formal specs to mobile apps. We should list them so that we can get a better understanding of the space.

Include:
- DID spec
- Verifiable claims
- Verifiable credentials
- DID-auth
- Blockstack
- uPort
- 3box
- ipfs-social-proofs

**Comment by r0kk3rz** [comment]
Hi, we have also been looking into this problem and have come up with the following as a solution. https://github.com/zippiehq/vault
https://github.com/zippiehq/vault-api 

It is a way to generate, use, and recover asymmetric  crypto keypairs entirely client side in the browser, useful for generating signatures for identity proofs.

We use this in our crypto currency wallet project, but we have plans to extend that to a general self sovereign identity system

**Comment by satazor** [comment]
@r0kk3rz interesting project, there's indeed similarities to what we are building. Could you explain a bit of what  https://github.com/zippiehq/amnesiacstore is and how it's used to share the master seed? Looking at the project code it's seems that it has some similarities to DID's Identity Hub, more specifically the inbox + storage part.

**Comment by r0kk3rz** [comment]
yeah we really need to document that part. It is a sort of 'identity hub' that we have as a simple restful web service for now, and we have some ideas about decentralising that further.

It has a few different interaction models, but they mostly revolve around providing a signed timestamp to perform an action (get or delete). We split the masterseed up using shamirs secret sharing and store part of it on amnesiacstore against a device specific key

**Comment by satazor** [comment]
Looking forward to see what you are building! I've added Vault to the list of similar projects. If you to share more details on what you are building or follow the process of IDM, you may attend the Weekly DDC meetings. Issue for the next meeting: https://github.com/ipfs/dynamic-data-and-capabilities/issues/72
